apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -11,18 +11,21 @@ import {
   nodeModulesAssetPath,
 } from '../../../lib/assets.js';
 import { b64EncodeUnicode } from '../../../lib/base64-util.js';
+import type { CopyTarget } from '../../../lib/copy-content.js';
 import { type AiQuestionGenerationPrompt, type Question } from '../../../lib/db-types.js';
 
 export function InstructorAiGenerateDraftEditor({
   resLocals,
   prompts,
   question,
   variantId,
+  questionCopyTargets,
```

Comment:
Why populate this at all on this page? When would you possibly have a publicly-shared draft question ([which in fact are forbidden from being shared](https://github.com/PrairieLearn/PrairieLearn/blob/46e423c74df2fa991e8d56dffebb1696465cb6b4/apps/prairielearn/src/sync/sharing.ts#L291-L295)) and want to copy it while you're iterating on it?

I see that you were just updating an existing implementation that was already using `setQuestionCopyTargets`. I'm proposing that we remove that from this page. Maybe we shift that to a separate PR too.

Reply:
I will admit I also had no idea why this was being put on the AI question generation page, I was just leaving things the way they were before. 

I will make a separate PR to do that.